src/admin/views/EditBannerView.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {Helmet} from 'react-helmet';
+import {useQuery, useMutation} from '@apollo/react-hooks';
+import {
+  Container,
+  Segment,
+  Header,
+  Message,
+  Dimmer,
+  Loader,
+} from 'semantic-ui-react';
+import {UPDATE_BANNER} from '../mutations';
+import {GET_BANNER, ALL_BANNERS} from '../queries';
+import WrappedBannerForm from '../forms/BannerForm';
```

Comment:
Why call this `WrappedBannerForm` and not just `BannerForm`?

Reply:
Haha I saw it somewhere else in the code and thought it was some sort of convention.